WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -1172,6 +913,75 @@ private void populateSelectedCategories() {
         mCategoriesTextView.setText(sb);
     }
 
+    private void setLocation(@Nullable Place place) {
+        if (mPostLocation == null) {
```

Comment:
This is going to be removed anyway, but it'd have been better if we did this after `if (place == null)` since we are nullifying the `mPostLocation` in that if anyway.